### DISCUSSION TITLE
"Failed to start Flutter renderer: Unable to create a GL context" on Linux AMD64 devices

### DISCUSSION THREAD

**Discussion by tux4fun** [rustdesk/rustdesk#7413] [Q&A] [answered]
I wanna notice you the following issue:

Splah screen of rustdesk-client don't pop-up, on Debian 12 Desktop amd 64. I reproduced each times the same error when you type in gnome-terminal : 

```
tux4fun@Indylight:~rustdesk 
(rustdesk:22162): Gtk-WARNING **: 13:03:42.827: Theme file for Graphite-dark-cursors has no directories
** (rustdesk:22162): WARNING **: 13:03:42.995: Failed to start Flutter renderer: Impossible de créer un contexte GL
```
```
tux4fun@Indylight:~$ uname -a && cat /etc/debian_version 
Linux Indylight 6.1.0-18-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.76-1 (2024-02-01) x86_64 GNU/Linux
12.5
```

"Failed to start Flutter renderer: Unable to create a GL context" on Linux AMD64 devices

My issue seems look like that post , anyway board (AMD64/ARM64): 

https://forum.odroid.com/viewtopic.php?t=46969

Thx.

**Comment by yank555-lu** [comment]
Exactly the same on my end, running Ubuntu 22.04.4 LTS updated to now (2024-04-18 10:42 CET), with KDE as window manager.

The listener part of rustdesk is working fine, I can control the host fine.

Initially I had v 1.2.3 of the client installed, but saw there is a 1.2.3-2 which refused to install. So I uninstalled 1.2.3 and installed 1.2.3-2 with no change.

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/5843
